### PR TITLE
Improve clarity of network proxy settings to include SOCKS4/5 proxy

### PR DESCRIPTION
**Pull request by hassanbenhzaine**
### Summary
This merge request add vsual improvements to the network proxy section, The changes contains added support for SOCKS4/5 proxy which is already supported by Insomnia but never mentioned in the UI.

Usufal links: 
- https://github.com/Kong/insomnia-docs/pull/149
- https://everything.curl.dev/usingcurl/proxies/socks

### Reason
Visual improvements take a look at this
https://github.com/Kong/insomnia/issues/1085

### Testing
I ran the project locally and everything seems to work, the changes are minor and can be tested by just reviwing the code.

### CODE REVIEW

File: packages/insomnia/src/ui/components/settings/general.tsx [hunk 1]
```diff
@@ -329,14 +329,16 @@ export const General: FC = () => {
 
       <div className="form-row pad-top-sm">
         <MaskedSetting
-          label='HTTP proxy'
+          label='Proxy for HTTP'
           setting='httpProxy'
+          help="Enter a HTTP or SOCKS4/5 proxy starting with appropriate prefix from the followihng (https://, socks4://, socks5://)"
```
Comment (filfreire) [inline packages/insomnia/src/ui/components/settings/general.tsx]:
```suggestion
          help="Enter a HTTP or SOCKS4/5 proxy starting with appropriate prefix from the following (http://, socks4://, socks5://)"
```

File: packages/insomnia/src/ui/components/settings/general.tsx [hunk 2]
```diff
@@ -329,14 +329,16 @@ export const General: FC = () => {
...
           placeholder="localhost:8005"
           disabled={!settings.proxyEnabled}
         />
         <MaskedSetting
-          label='HTTPS proxy'
+          label='Proxy for HTTPS'
           setting='httpsProxy'
+          help="Enter a HTTPS or SOCKS4/5 proxy starting with appropriate prefix from the followihng (https://, socks4://, socks5://)"
```
Comment (filfreire) [inline packages/insomnia/src/ui/components/settings/general.tsx]:
```suggestion
          help="Enter a HTTPS or SOCKS4/5 proxy starting with appropriate prefix from the following (https://, socks4://, socks5://)"
```

**Review by filfreire — COMMENTED** [review]
Thanks for contributing @hassanbenhzaine. Left some comments with possible fixes.

**Comment by hassanbenhzaine** [comment]
@filfreire I applied the fixes you commented :)

**Review by filfreire — APPROVED** [review]

**Pull request merged**